src/goose/command/base.py

```diff
@@ -8,9 +8,20 @@ class Command(ABC):
     """A command that can be executed by the CLI."""
 
     def get_completions(self, query: str) -> List[Completion]:
-        """Get completions for the command."""
+        """
+        Get completions for the command.
+
+        Args:
+            query (str): The current query.
+        """
         return []
 
-    def execute(self, query: str) -> Optional[str]:
-        """Execute's the command and replaces it with the output."""
+    def execute(self, query: str, surrounding_context: str) -> Optional[str]:
+        """
+        Execute's the command and replaces it with the output.
+
+        Args:
+            query (str): The query to execute.
+            surrounding_context (str): The full user message that the query is a part of.
+        """
         return ""
```

Comment:
why does this return empty string?

Reply:
It a command should always return a string, but folks may build the completions portion of their command before building the execution part! Returning a string here means that folks will have a decent experience building commands (kinda like [here in the base toolkit](https://github.com/square/goose/blob/dd126afa6c73fe01ad4991a6b45a6d027cb02b29/src/goose/toolkit/base.py#L54))

Reply:
oh i didn't see this was the base class. thanks!!